callback.go

```diff
@@ -32,6 +34,10 @@ type CallbackInfo struct {
 	// function, if any. When used in conjunction with [RecoverFromPanics],
 	// this will be set to a [PanicError] when the function panics.
 	Error error
+
+	// Values contains all values constructed by the [Callback]'s
+	// associated function.
```

Comment:
Given the discussion, let's add some explicit documentation here that mentions the undefined results of modifying these values.

Reply:
I updated the comment!